src/main/java/frc/robot/subsystems/pivot/PivotIOTalon.java

```diff
@@ -0,0 +1,216 @@
+            PivotConstants.MAX_ACCELERATION_ROTATIONS_PER_SECOND_SQUARED;
+
+        pivotConfig.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+        pivotConfig.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+        pivotConfig.MotorOutput.DutyCycleNeutralDeadband = HardwareConstants.MIN_FALCON_DEADBAND;
+
+        pivotConfig.ClosedLoopGeneral.ContinuousWrap = true;
+
+        pivotConfig.Feedback.FeedbackSensorSource = FeedbackSensorSourceValue.RemoteCANcoder;
+        pivotConfig.Feedback.FeedbackRemoteSensorID = pivotEncoder.getDeviceID();
+
+        pivotConfig.SoftwareLimitSwitch.ForwardSoftLimitThreshold = PivotConstants.MAX_ANGLE;
+        pivotConfig.SoftwareLimitSwitch.ReverseSoftLimitThreshold = PivotConstants.MIN_ANGLE;
+        pivotConfig.SoftwareLimitSwitch.ForwardSoftLimitEnable = true;
+        pivotConfig.SoftwareLimitSwitch.ReverseSoftLimitEnable = true;
+        leaderPivotMotor.getConfigurator().apply(pivotConfig, HardwareConstants.TIMEOUT_S);
+        pivotConfig.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+        followerPivotMotor.getConfigurator().apply(pivotConfig, HardwareConstants.TIMEOUT_S);
+
+
+        leaderPivotMotor.getConfigurator().apply(pivotConfig, 1.0);
+        followerPivotMotor.getConfigurator().apply(pivotConfig, 1.0);
+        leaderPivotMotor.getConfigurator().apply(controllerConfig, 1.0);
+        followerPivotMotor.getConfigurator().apply(controllerConfig, 1.0);
+
+        leaderPosition = leaderPivotMotor.getPosition();
+        leaderVelocity = leaderPivotMotor.getVelocity();
+        leaderAppliedVolts = leaderPivotMotor.getMotorVoltage();
+        leaderSupplyCurrent = leaderPivotMotor.getSupplyCurrent();
+        leaderTorqueCurrent = leaderPivotMotor.getTorqueCurrent();
+        leaderTempCelsius = leaderPivotMotor.getDeviceTemp();
+        followerPosition = followerPivotMotor.getPosition();
+        followerVelocity = followerPivotMotor.getVelocity();
+        followerAppliedVolts = followerPivotMotor.getMotorVoltage();
+        followerSupplyCurrent = followerPivotMotor.getSupplyCurrent();
+        followerTorqueCurrent = followerPivotMotor.getTorqueCurrent();
+        followerTempCelsius = followerPivotMotor.getDeviceTemp();
+        
+        
+        pivotPos = pivotEncoder.getAbsolutePosition();
+
+        BaseStatusSignal.setUpdateFrequencyForAll(
+            100.0,
+            leaderPosition,
+            leaderVelocity,
+            leaderAppliedVolts,
+            leaderSupplyCurrent,
+            leaderTorqueCurrent,
+            leaderTempCelsius,
+            followerPosition,
+            followerVelocity,
+            followerAppliedVolts,
+            followerSupplyCurrent,
+            followerTorqueCurrent,
+            followerTempCelsius,
+            pivotPos);
+    }
+
+    @Override
+    public void updateInputs(PivotIOInputs inputs){
+        BaseStatusSignal.refreshAll(
```

Comment:
this is not how you do `updateInputs()`. It should look similar to how you did it in PivotIOSim. As it is now you won't be logging anything as you haven't done any of the `inputs.input = motorSignal` stuff, like yall did in the sim impl.